changelogs/8.2.asciidoc

```diff
@@ -0,0 +1,43 @@
+[[release-notes-8.2]]
+== APM version 8.2
+
+https://github.com/elastic/apm-server/compare/8.1\...8.2[View commits]
+
+* <<release-notes-8.2.0>>
+
+[float]
+[[release-notes-8.2.0]]
+=== APM version 8.2.0
+
+https://github.com/elastic/apm-server/compare/8.1.3\...8.2.0[View commits]
+
+[float]
+==== Breaking Changes
+- APM Server now emits events with `event.duration` and renames the field to `<event>.duration.us` in an ingest pipeline {pull}7261[7261]
+- Removed `observer.version_major` {pull}7399[7399]
+- APM Server no longer ships with SystemV init scripts or the go-daemon wrapper; use systemd {pull}7576[7576]
+
+[float]
+==== Deprecations
+- APM Server support for 32-bit architectures has been deprecated and will be removed in a future release {pull}7678[7678]
+
+[float]
+==== Bug fixes
+- Do not overwrite `service version` if no transaction/error/... specific `service.name` is given {pull}7281[7281]
+- Set transaction name to `"<method> unknown route"` for unknown HTTP routes in APM Server instrumentation {pull}7679[7679]
+- Fix a bug that caused the APM Server to run out of memory it receives events from a high number of APM agents {pull}7809[7809]
```

Comment:
We shipped this bug fix on `8.1.3` do we want to still reference it here?

Reply:
Thanks for catching that. This should be removed.